File: PYTHON/TP3/src/etudiant.py
from src.personne import Personne

class Etudiant(Personne):
    def __init__(self, nom, age, numero_etudiant, moyenne, cours):
        super().__init__(nom, age)
        self.numero_etudiant = numero_etudiant
        self.moyenne = moyenne
        self.cours = cours

File: TP1/JAVA/TP1/src/Etudiant.java
package src;
import java.util.ArrayList;
import java.util.List;

public class Etudiant extends Personne {
    public String numeroEtudiant;
    public double moyenne;
    public List<Cours> listeCours;

    public Etudiant(String nom, int age, String numeroEtudiant, double moyenne) {
        super(nom, age);
        this.numeroEtudiant = numeroEtudiant;
        this.moyenne = moyenne;
        this.listeCours = new ArrayList<>();
    }

    public void ajouterCours(Cours cours) {
        this.listeCours.add(cours);
    }
}